src/blob/index/bptree/tests.rs

```diff
@@ -65,12 +65,29 @@ impl Into<usize> for KeyType {
     }
 }
 
+fn get_test_dir(dir_name: &str) -> PathBuf {
+    let result = std::env::temp_dir()
+        .join("pearl_bptree_test")
+        .join((std::time::UNIX_EPOCH.elapsed().unwrap().as_secs() % 1_563_100_000).to_string())
```

Comment:
This is not a constant. It is the time in seconds elapsed from `UNIX_EPOCH`. It should be calculated for each test run to avoid running on top of previous run results

Reply:
I mean `1_563_100_000`

Reply:
That is not important constant. I copied initialization from `tests/common.rs`. My guess, is that it was chosen to reduce the length of the string representation of the number (current UNIX timestamp is `1683232306`).  I think I'll just remove it